recipes/ozo/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20210509":
+    url: "https://github.com/yandex/ozo/archive/1b06a00ec3cea09108557bbec71cc7a6455dfb6b.zip"
```

Comment:
We prefer the tar.gz since they are smaller

Reply:
Generally, for everything I have not answered: I'll try and apply everything tomorrow.